resources/views/admin/users.blade.php

```diff
@@ -0,0 +1,50 @@
+@extends('admin/layout')
+@section('content')
+
+<!DOCTYPE html>
```

Comment:
this will render double html header as admin/layout already contains html headers